images/linux/scripts/installers/git-ftp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+################################################################################
+##  File:  git-ftp.sh
+##  Desc:  Installs git-ftp
+################################################################################
```

Comment:
Do we really need the separate file for this tool? installation is pretty small and simple. I think it makes sense to include these changes to `git.sh`

Reply:
or even to basic.sh?

Reply:
Move it to git.sh